src/pages/patientView/clinicalInformation/PatientViewPageStore.ts

```diff
@@ -740,6 +791,26 @@ export class PatientViewPageStore {
         invoke: async () => client.getStudyUsingGET({ studyId: this.studyId }),
     });
 
+    public sampleManager = remoteData<SampleManager>({
```

Comment:
do we not already have a sampleManager on patient page somewhere?  should basically be a singleton unless there's something i'm missing.

Reply:
Actually I copied it here so I can use it in structural variant table component. I forgot to remove it from patient page